tests/baselines/reference/typeReferenceDirectives10.js

```diff
@@ -0,0 +1,26 @@
+//// [tests/cases/compiler/typeReferenceDirectives10.ts] ////
+
+//// [ref.d.ts]
+
+export interface $ { x }
+
+//// [index.d.ts]
+declare let $: { x: number }
+
+//// [app.ts]
+/// <reference types="lib"/>
+import {$} from "./ref";
+export interface A {
+    x: $
```

Comment:
can you add another test, where `$` is used as a value, as that would resolve to the `let $` and should  the `lib`.